README.md

```diff
@@ -150,3 +150,7 @@ The format is `<type>(<scope>): #<issue-no> <subject>`, examples...
 1. `set GOOGLE_APPLICATION_CREDENTIALS=path\to\key.json`
 2. `firebase setup:emulators:firestore`
 3. `firebase emulators:start`
+
+## Generate documentation
```

Comment:
Having a blank line after `headers` is consistent with the rest of the file and easier to read

Reply:
done